cmd/tk/toolCharts.go

```diff
@@ -143,6 +147,36 @@ func chartsInitCmd() *cli.Command {
 	return cmd
 }
 
+func chartsVersionCheckCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "version-check",
+		Short: "Check required charts for updated versions",
+	}
+	repoConfigPath := cmd.Flags().String("repository-config", "", repoConfigFlagUsage)
+
+	cmd.Run = func(cmd *cli.Command, args []string) error {
+		c, err := loadChartfile()
+		if err != nil {
+			return err
+		}
+
+		data, err := c.VersionCheck(*repoConfigPath)
+		if err != nil {
+			return err
+		}
+
+		returnData, err := json.Marshal(data)
+		if err != nil {
+			return err
+		}
+
+		fmt.Printf("%s", returnData)
+		return nil
```

Comment:
I also stuck the indent option behind a `--pretty-print` flag in order to give users the option since some tools are easier to work with if the output is one line.